projects/storefrontlib/src/lib/cart/add-to-cart/add-to-cart.component.html

```diff
@@ -4,7 +4,7 @@
   [disabled]="quantity <= 0"
   (click)="addToCart()"
 >
-  <span *ngIf="(cartEntry$ | async) as entry" class="entry-quantity">{{
+  <span *ngIf="(cartEntry$ | async) as entry" class="cx-entry-quantity">{{
```

Comment:
I think this can be removed as it's not present anywhere. There probably wont be any side effects on the dev's end. As this isn't in the scope of the refactoring we should create a ticket to address this unless UX says otherwise.

Reply:
Removed